src/components/PositionCard/index.tsx

```diff
@@ -157,17 +158,20 @@ export function MinimalPositionCard({ pair, showUnwrapped = false, border }: Pos
   )
 }
 
-export default function FullPositionCard({ pair, border }: PositionCardProps) {
+export default function FullPositionCard({ pair, border, stakedBalance }: PositionCardProps) {
   const { account } = useActiveWeb3React()
 
   const currency0 = unwrappedToken(pair.token0)
   const currency1 = unwrappedToken(pair.token1)
 
   const [showMore, setShowMore] = useState(false)
 
-  const userPoolBalance = useTokenBalance(account ?? undefined, pair.liquidityToken)
+  const userDefaultPoolBalance = useTokenBalance(account ?? undefined, pair.liquidityToken)
   const totalPoolTokens = useTotalSupply(pair.liquidityToken)
 
+  // if staked balance balance provided, use instead of fetched balance from pool
+  const userPoolBalance = stakedBalance ?? userDefaultPoolBalance
```

Comment:
shouldn't it be the combination?

Reply:
Guess its a UX question - i could see the case for showing the total, and then giving 3 buttons (add, remove, manage staked)

Reply:
and then a line for the staked amount?

Reply:
Updated to a combination, with an extra button and row of data if theres a staked amount